src/parser/parser_driver.cpp

```diff
@@ -441,7 +441,21 @@ void ParserDriver::defineGrammar()
 				return {};
 			},
 			"tags", "LCB", "metas", "strings", "condition", "RCB", [&](auto&& args) -> Value {
-				std::optional<TokenIt> mod = std::move(args[0].getOptionalTokenIt());
+				std::optional<TokenIt> mod_private = {};
+				std::optional<TokenIt> mod_global = {};
+				const std::vector<TokenIt> mods = std::move(args[0].getMultipleTokenIt());
+				for (const auto &token: mods) {
```

Comment:
Following the used code-style in yaramod, I would rather put the curly bracket `{` on a new line.